doc/source/example_notebooks/IS2_cloud_data_access.ipynb

```diff
@@ -12,8 +12,16 @@
     "## Notes\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ipx.Variables(path=gran_ids[1][0]).avail()
</pre><p>This cell errors if you've run the above two cells (not sure if it does on it's own).</p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [8], line 1
----&gt; 1 ipx.Variables(path=gran_ids[1][0]).avail()

File ~/icepyx/icepyx/core/variables.py:95, in Variables.__init__(self, vartype, path, product, version, avail, wanted, auth)
     93         auth = None
     94     # Read the product and version from the file
---&gt; 95     self._product = is2ref.extract_product(self._path, auth=auth)
     96     self._version = is2ref.extract_version(self._path, auth=auth)
     97 elif product:
     98     # Check for valid product string

File ~/icepyx/icepyx/core/is2ref.py:365, in extract_product(filepath, auth)
    361         raise AttributeError(
    362             "Must provide credentials to auth if accessing s3 data"
    363         )
    364     # Read the s3 file
--&gt; 365     s3 = earthaccess.get_s3fs_session(daac="NSIDC", provider=auth)
    366     f = h5py.File(s3.open(filepath, "rb"))
    367 else:
    368     # Otherwise assume a local filepath. Read with h5py.

File /srv/conda/envs/notebook/lib/python3.10/site-packages/earthaccess/api.py:337, in get_s3fs_session(daac, provider, results)
    326 """Returns a fsspec s3fs file session for direct access when we are in us-west-2
    327 
    328 Parameters:
   (...)
    334     class s3fs.S3FileSystem: an authenticated s3fs session valid for 1 hour
    335 """
    336 daac = _normalize_location(daac)
--&gt; 337 provider = _normalize_location(provider)
    338 if results is not None:
    339     endpoint = results[0].get_s3_credentials_endpoint()

File /srv/conda/envs/notebook/lib/python3.10/site-packages/earthaccess/api.py:27, in _normalize_location(location)
     17 """Handle user-provided daac and provider values
     18 
     19 These values must have a capital letter as the first character
   (...)
     24 https://wiki.earthdata.nasa.gov/display/ED/CMR+Data+Partner+User+Guide?src=contextnavpagetreemode
     25 """
     26 if location is not None:
---&gt; 27     location = location.upper()
     28 return location

AttributeError: 'Auth' object has no attribute 'upper'
</pre><p>I'm also getting the same auth error when I try to run read later on in the notebook...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/468/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
@JessicaS11 When I "Run All" cells in this notebook everything goes smoothly. Can you perhaps describe with some more detail how you got the error? Are you authenticating with a .netrc, manual entry, or env variables?

Reply:
@rwegener2 I'm authenticating manually and was running the notebook in CryoCloud. Unfortunately, in trying to reproduce again just now I've instead spent the last 30 minutes trying to get the notebook to install a dev version of icepyx witihin CryoCloud (again), to no avail. However, I did try it locally for fun and was surprised I got a permissions error, which I wasn't expecting since for this line it should only need my normal earthdata login, right?

Reply:
What version of earthdata are you running? I'll look into that as a potential error source if I ever get CryoCloud install/import the right version of icepyx.

Reply:
Thanks for digging into this @JessicaS11. I have earthaccess 0.5.2 installed in my dev env. Sounds like what needs to happen is a change in syntax, which seems doable.